src/Sulu/Bundle/AdminBundle/Resources/translations/admin.en.json

```
@@ -31,6 +31,7 @@
     "sulu_admin.no": "No",
     "sulu_admin.type": "Type",
     "sulu_admin.default": "Default",
+    "sulu_admin.locale": "Sprache",
```

`locale` I think is uncommon and maybe we go with `:

```suggestion
    "sulu_admin.language": "Sprache",
```